core/src/main/scripts/importer/validateData.py

```diff
@@ -268,9 +268,9 @@ def load_genome_info(self, properties_filename):
                 sp_line = line.split('=', 1)
                 if sp_line[0] == 'species':
                     self.species = sp_line[1]
-                elif sp_line[0] == 'ncbi.build':
-                    self.ncbi_build = sp_line[1]
-                elif sp_line[0] == 'ucsc.build':
+                elif sp_line[0] == 'genome.build':
+                    self.ncbi_build = sp_line[1].split(",")
```

Comment:
maybe rename this variable to `genome_builds` ?

Reply:
agree! will make changes.